### ISSUE TITLE
Protection Scheme Signaling in MPD

### ISSUE THREAD

**Issue by tinskip** [Dash-Industry-Forum/CPIX#49]
Currently there is no signaling for the protection scheme in the ContentProtection element.  This info is needed by the application to figure out whether streams can be played on the device.  We may want to carry multiple ContentProtection elements in DrmSystem for multiple scheme support.

(Perhaps out of scope, but could not find a better place).

**Comment by arjen4157** [comment]
That depends whether the ContentProtection Element is part of the signaled data. The specification seems to imply that this is not the case, but looking at the example clearKey.xml file it *is* part of the ContentProtectionData.

I think it should be present; since it includes the @schemeIdUri, @value. It also allows us to e.g. define an additional namespace. E.g. in the case of:
<ContentProtection xmlns:mspr="urn:microsoft:playready" schemeIdUri="urn:uuid:9A04F079-9840-4286-AB92-E65BE0885F95" value="MSPR 2.0">
    <mspr:pro>  </mspr:pro>
</ContentProtection>

**Comment by arjen4157** [comment]
Hmm. Posting XML snippets doesn't work well. Let me try again:

--ContentProtection xmlns:mspr="urn:microsoft:playready" schemeIdUri="urn:uuid:9A04F079-9840-4286-AB92-E65BE0885F95" value="MSPR 2.0"--
    --mspr:pro--  --/mspr:pro--
--/ContentProtection--

**Comment by tinskip** [comment]
I was referring to the CENC protection scheme.  I.E 'cenc', 'cbc1', 'cens', 'cbcs'.

**Comment by arjen4157** [comment]
Sorry, I thought you meant the DRM scheme/system. My bad. I've deleted the comment and raised a new issue instead (https://github.com/Dash-Industry-Forum/CPIX/issues/50).

**Comment by lpiron** [comment]
This is for the signaling in the MPD if I understand correctly. There was a proposal made by Iraj at MPEG, a draft is available on DASH-IF (http://files.groupspaces.com/dashif/files/1938908/mxxxxx-DASHProfileIOPSignaling.docx). The MPEG contribution is m41175. It covers video/audio/subtitles, maybe we should extend it with encryption method. Is it what you have in mind ?

**Comment by lpiron** [comment]
IOP #194 is closed, hence this one also
